### PR TITLE
Clickable email stats feature

### PR DESCRIPTION
**Pull request by virlatinus**
| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Related user documentation PR URL | N/A
| Related developer documentation PR URL | N/A
| Issues addressed (#s or URLs) | N/A
| BC breaks? | N/A
| Deprecations? | N/A

#### Description:
This PR makes it possible to click on an email stat and see a list of contacts that the email was sent to, that are pending or queued to send or that read the email. 

#### Steps to test this PR:
1. Apply this PR
2. Go to Channels/Emails
3. Click on any stat to see the list of contacts for that specific stat

### CODE REVIEW

**Comment by npracht** [comment]
Awesome!!! What about clickers, unsubscribers and bounces?!

**Comment by luizeof** [comment]
\o/
+1

**Comment by npracht** [comment]
Hello @virlatinus !
Did you have time to consider my message ?
>Awesome!!! What about clickers, unsubscribers and bounces?!

**Comment by virlatinus** [comment]
Hi @npracht. We are going to start with emails sent, read, queued and pending. But those sound like good additions also. We'll see if we can include them in this PR.

**Comment by npracht** [comment]
Thanks for your reply @virlatinus !
It would be very great, we know that when it is left aside it often get forgotten :-)

And you also may know that clickers are the hottest leads and consenquently much mor important for a marketer than the sent 👼

File: app/bundles/EmailBundle/Views/Email/list.html.php [hunk 1]
```diff
@@ -161,32 +161,54 @@
...
-                            </div>
-                        </span>
-                        <span class="mt-xs label label-primary" id="read-percent-<?php echo $item->getId(); ?>">
-                            <div style="width: 50px;">
-                                <i class="fa fa-spin fa-spinner"></i>
-                            </div>
-                        </span>
-                        <?php echo $view['content']->getCustomContent('email.stats', $mauticTemplateVars); ?>
+                        <div class="btn-group btn-group-xs">
+                            <button class="mt-xs btn btn-default hide"
```
Comment (alanhartless) [inline app/bundles/EmailBundle/Views/Email/list.html.php]: Making these buttons use up a lot of room. Can we use a tags inside the span labels instead and use CSS to change the color to white and on hover to white then add the class `has-click-event`

Comment (virlatinus) [inline app/bundles/EmailBundle/Views/Email/list.html.php]: I did what you suggested Alan.

**Comment by luizeof** [comment]
@virlatinus can you show us some screen ?

Thanks for your hard work!

**Comment by virlatinus** [comment]
@luizeof 
image

image

**Comment by khaoua** [comment]
#4877

**Comment by ddaniele18** [comment]
Just new and getting into Mautic. Has this issue (being able to see contacts who click on links in an email) been resolved or is there a good work around? This is super important to us but I haven't found a work around to use. Thanks.

**Pull request merged**